src/app/components/LiveLabel/types.ts

```diff
@@ -0,0 +1,18 @@
+export interface LiveLabelProps {
+  offScreenText?: string;
+  lang?: string;
+  id?: string;
+  className?: string;
+}
+
+export interface TextProps {
+  offScreenText?: string;
+  lang?: string;
+  id?: string;
+}
```

Comment:
Perhaps this is overkill, but could LiveLabel props extend TextProps, so that we don't need to duplicate the types?

```
export interface LiveLabelProps extends TextProps {
  className?: string
}
```

Reply:
All done